Description: FindJNI: Add ppc64le architecture
Author: Dinar Valeev <dvaleev@suse.com>
Origin: upstream, commits ac2527e1d988afa18e141a660ec542b7a3f7f352 and
 18ac6713e8657cca8fc791bcd581a47a7be6ebce

diff --git a/Modules/FindJNI.cmake b/Modules/FindJNI.cmake
index 669e3e2..6a496af 100644
--- a/Modules/FindJNI.cmake
+++ b/Modules/FindJNI.cmake
@@ -54,6 +54,8 @@ macro(java_append_library_directories _var)
         # mips* machines are bi-endian mostly so processor does not tell
         # endianess of the underlying system.
         set(_java_libarch "${CMAKE_SYSTEM_PROCESSOR}" "mips" "mipsel" "mipseb" "mips64" "mips64el" "mipsn32" "mipsn32el")
+    elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(powerpc|ppc)64le")
+        set(_java_libarch "ppc64" "ppc64le")
     elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(powerpc|ppc)64")
         set(_java_libarch "ppc64" "ppc")
     elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(powerpc|ppc)")
